chia/full_node/weight_proof.py

```diff
@@ -707,7 +707,7 @@ def _create_sub_epoch_data(
 ) -> SubEpochData:
     reward_chain_hash: bytes32 = sub_epoch_summary.reward_chain_hash
     #  Number of subblocks overflow in previous slot
-    previous_sub_epoch_overflows = uint8(sub_epoch_summary.num_blocks_overflow)  # total in sub epoch - expected
+    previous_sub_epoch_overflows: uint8 = sub_epoch_summary.num_blocks_overflow  # total in sub epoch - expected
```

Comment:
If it is already `uint8`, is the hint needed?  As a (slightly) bigger adjustment, looks like we could just not have this (any of these?) variables at this point and just pass the attributes straight to the `SubEpochData()` call.  Take on whichever bits of this you feel like and feel free to close the conversation.